gap/ideals/ideals.gi

```diff
@@ -187,81 +187,121 @@ InstallMethod(Representative, "for a semigroup ideal",
 
 # a convenience, similar to the functions <Semigroup>, <Monoid>, etc
 
-InstallGlobalFunction(SemigroupIdeal,
-function(arg...)
+InstallGlobalFunction(AnySemigroupIdealInputParsing,
+function(inputArgs)
```

Comment:
Might be better to stick to `arg...` and whenever you want to call this function do:

~~~
CallFuncList(AnySemigroup..., arg);
~~~

this is like doing 
~~~
AnySemigroup...(*arg)
~~~
in python